packages/core/src/physics/CharacterController.ts

```
@@ -10,13 +10,28 @@ import { ColliderShape } from "./shape";
  * The character controllers.
  */
 export class CharacterController extends Collider {
-  private _stepOffset: number = 0.5;
+  private _stepOffset = 0.5;
   private _nonWalkableMode: ControllerNonWalkableMode = ControllerNonWalkableMode.PreventClimbing;
   private _upDirection = new Vector3(0, 1, 0);
-  private _slopeLimit: number = 0.707;
+  private _slopeLimit = 0.707;
```

Biome

[error] Prefer constants from the standard library.

Unsafe fix: Use Math.SQRT1_2 instead.


(lint/suspicious/noApproximativeNumericConstant)